.github/workflows/kind-e2e.yaml

```diff
@@ -31,11 +31,11 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         pipelines-release:
-        - v0.44.4  # LTS
-        - v0.47.3  # LTS
-        - v0.50.1  # LTS
-        - v0.51.0
+        - v0.50.5  # LTS
+        - v0.53.5  # LTS
+        - v0.56.3  # LTS
+        - v0.59.2  # LTS
```

Comment:
I think we should have `v0.61.0` in the list as that is the latest one.

Reply:
Yeah, but if you see the end of life, it is already gone, hence I did not add

v0.61

    Latest Release: [v0.61.0](https://github.com/tektoncd/pipeline/releases/tag/v0.61.0) (2024-06-25) ([docs]
    End of Life: 2024-07-25


Shall we still add ?

Reply:
Missed that. Let's not add it then.